docker-run

```diff
@@ -30,8 +30,8 @@ import tempfile
 
 def main(args):
     args.volume = ('-v {}:/home/iot-lab-gateway/gateway_code'
-                   .format(os.path.abspath(args.volume)
-                           if args.volume is not None else ''))
+                   .format(os.path.abspath(args.volume))
```

Comment:
oups, good find !

Reply:
easy find, docker-run crashes because we'd be passing `-v :/home/iot-lab-gateway/gateway_code` to `docker run` :-)